src/test/ui/parser/issue-103425.rs

```diff
@@ -0,0 +1,7 @@
+fn f() -> f32 {
+    3
+    //~^ ERROR expected `;`
+    5.0
+}
```

Comment:
Do you mind adding the other test where we have several missing semicolons? https://github.com/rust-lang/rust/issues/103425#issuecomment-1288121143

```
fn f() -> f32 {
    2_u32
    3_i8
    5.0
}

fn main() {
}
```

Reply:
Added it.